spec/lib/danger/ci_sources/support/pull_request_finder_spec.rb

```diff
@@ -103,5 +103,29 @@ def open_pull_requests_info
         expect(result.base).to eq "pr 518 base commit sha1"
       end
     end
+
+    context "specify api endpoint of octokit client" do
+      it "By DANGER_GITHUB_API_HOST" do
+        ENV["DANGER_GITHUB_API_HOST"] = "https://enterprise.artsy.net"
+
+        allow(Octokit::Client).to receive(:new).with(
```

Comment:
If I recall - Allow is used to say "mock theses upcoming functions" basically, and expect says "perform upcoming expectations on this object"